### PR TITLE
Add action steps outline for community chapter

### PR DESCRIPTION
**Pull request by mbonsma**
Adding a bullet-point list of the topics and code concepts that will be covered in this chapter. Once all these points are covered, the final exercises are fully doable. 

This chapter is very description-heavy and code-light, so I included links to existing materials that I think do a good job of covering the concepts we want to address. No need to reinvent the wheel when writing time comes!

The one remaining grey area in both the final exercises and the action steps is what we want to cover under "Project Governance". I included some suggestions, but this is open for discussion.

### CODE REVIEW

**Comment by lwjohnst86** [comment]
First out of the gate! :partying_face: :clap: :raised_hands:

File: chapters/10-community-involvement.Rmd [hunk 1]
```diff
@@ -9,7 +9,28 @@
...
+  - Code: use the `reprex` package to create a minimal reproducible example
+  - Text: using a NEWS file ([reference](https://r-pkgs.org/release.html#news))
+  - Code: `usethis::use_news_md()`
+- Code of Conduct
+  - Text: inclusivity, safety, the purpose of a Code of Conduct and what goes into one ([reference](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/)).
+  - Code: `usethis::use_code_of_conduct()`
+- Contributing guidelines
+  - Text: the purpose of Contributing guidelines and what they can contain ([reference](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/)). Emphasize that the different sections are quite project-specific (i.e. your project might not use communication channels outside of GitHub).
+  - Code: `usethis::use_tidy_contributing()`
+- Licensing
```
Comment (k8hertweck) [inline chapters/10-community-involvement.Rmd:29]: I think we should move licensing up to above contributing guidelines; licenses are for folks to know how to use AND contribute to software, so then the flow of the chapter would continue to narrow down in scope.

File: chapters/10-community-involvement.Rmd [hunk 2]
```diff
@@ -9,7 +9,28 @@
 - NEWS file
 - reprex
 
+## Outline of action steps
 
+This chapter is very description-heavy with minimal code.
+
+- Creating a welcoming community with welcoming materials
+<!-- I grouped reprex and NEWS with the README because they seem related to the general welcoming presentation of a project, but they could go elsewhere. -->
```
Comment (k8hertweck) [inline chapters/10-community-involvement.Rmd:17]: I initially waffled about where to place NEWS and reprex, but I think the way you've framed it here is great. I am wondering about the interactions between this README explanation and other places where we discuss documentation?

**Review by k8hertweck — COMMENTED** [review]
I'm thinking about this more as I continue my notes on the flow of the entire book. Right now I think it's great to have this amount of detail identified! This chapter connects with lots of other places and can be tricky.

File: chapters/10-community-involvement.Rmd (hunk 2 above)
Comment (mbonsma) [reply chapters/10-community-involvement.Rmd:17]: True, right now README is first introduced in [chapter 8](https://github.com/merely-useful/r-rse/blob/main/chapters/08-documentation.Rmd) I believe. Maybe in this chapter the focus should be more about how to make an existing README welcoming, whereas in chapter 8 it's about putting in a README right from the beginning.

File: chapters/10-community-involvement.Rmd [hunk 3]
```diff
@@ -9,7 +9,28 @@
...
+  - Text: the purpose of a README and what goes into one ([Mozilla reference](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), [R packages reference](https://r-pkgs.org/release.html#readme)).
```
Comment (mbonsma) [inline chapters/10-community-involvement.Rmd:18]: @k8hertweck, how about this more specific language around modifying a README to be welcoming? 

```suggestion
  - Text: making your README welcoming and easy to understand ([Mozilla reference](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), [R packages reference](https://r-pkgs.org/release.html#readme)).
```

File: chapters/10-community-involvement.Rmd (hunk 2 above)
Comment (lwjohnst86) [reply chapters/10-community-involvement.Rmd:17]: Definitely agree!

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**